### PR TITLE
Fixed problem with path to tribits install

### PR DESCRIPTION
**Pull request by agsalin**
${PROJECT_NAME} is not defined, so the DESTINATION variable
was set to "/cmake" where I do not have write permission.

${CMAKE_PROJECT_NAME} is however defined.

### CODE REVIEW

**Comment by agsalin** [comment]
Ross -- Trilinos failed to install for me because of this issue. Seems to be a lot of use PROJECT_NAME, while this is undefined for me. So, my fix is enough to get my build working again but is not the whole issue.

**Comment by rppawlo** [comment]
Drekar CI is also failing with the same issue

**Comment by bartlettroscoe** [comment]
There is a fix on the release branch but I must have merged for release before I created this fix.  I will merge again and it should fix this.

File: cmake/CallbackSetupExtraOptions.cmake [hunk 1]
```diff
@@ -6,7 +6,7 @@ SET(CMAKE_MODULE_PATH  ${CMAKE_MODULE_PATH} "${Trilinos_SOURCE_DIR}/cmake")
 # Install TriBITS so that other projects can use it.
 INSTALL(
   DIRECTORY "${Trilinos_SOURCE_DIR}/cmake/tribits"
-  DESTINATION "${${PROJECT_NAME}_INSTALL_LIB_DIR}/cmake"
+  DESTINATION "${${CMAKE_PROJECT_NAME}_INSTALL_LIB_DIR}/cmake"
```
Comment (bartlettroscoe) [inline cmake/CallbackSetupExtraOptions.cmake]: Does this really fix this?  I have a fix about to push (that was on the release branch already).  Not sure how that happened.

**Comment by bartlettroscoe** [comment]
This should be resolved with the commit  1176a8d9afd490a0dc61806037d6a5926b0ad65a that I just pushed to 'master'.  Please let me know.

**Comment by agsalin** [comment]
Yes, this fixed it. Thanks.

**Comment by bartlettroscoe** [comment]
> Yes, this fixed it. Thanks. 

Sorry again.  I am closing this PR.

**Pull request closed without merging**